app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<% provide(:title, "Edit Album")%>
```

Comment:
This is incredibly boss! I remember coming across this issue & using `provide` is a super clever way to address it. Nice job!